lib/mindwendel/local_storage.ex

```diff
@@ -0,0 +1,70 @@
+defmodule Mindwendel.LocalStorage do
+  alias Mindwendel.Permissions
+
+  @moduledoc """
+  The LocalStorage context. It includes helpers to handle local storage data from clients.
+  """
+
+  # This function is used to merge the brainstormings from local storage and the session.
+  # It returns a list of brainstormings sorted by the last_accessed_at field.
+  def brainstormings_from_local_storage_and_session(
+        brainstormings_from_local_storage,
+        brainstormings_from_session,
+        user
+      ) do
+    (brainstormings_from_local_storage(brainstormings_from_local_storage) ++
+       brainstormings_from_session(brainstormings_from_session, user))
+    |> Enum.uniq_by(& &1["id"])
+    |> Enum.sort(&(&1["last_accessed_at"] > &2["last_accessed_at"]))
```

Comment:
there is also `Enum.sort_by` :) https://hexdocs.pm/elixir/1.17.2/Enum.html#sort_by/3